packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -1107,7 +1113,33 @@ const ReactDataGrid = createReactClass({
           </div>
         </div>
       );
+  },
+
+  metricsUpdated() {
+    let columnMetrics = this.createColumnMetrics();
+
+    if (this.props.groups) {
```

Comment:
lets create a helper for this `usingHeaderGroups`, just in case we need to change what that check actually is